### PR TITLE
Added login & register presentation components 

### PR DESCRIPTION
**Pull request by evelan**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
     <link href="/css/heroic-features.css" rel="stylesheet">
     <link href="/css/modern-business.css" rel="stylesheet">
     <link href="/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+
```
Comment (kkorus) [inline public/index.html]: We can remove blnak lines :)

File: src/components/HomeComponent.js [hunk 2]
```diff
@@ -1,6 +1,28 @@
 import React from 'react';
+import firebase from 'firebase';
 
 export default class Home extends React.Component {
+  constructor(props) {
+      super(props);
+      this.state = {
```
Comment (kkorus) [inline src/components/HomeComponent.js]: State initialization in ctor or in getInitialState method?

Comment (evelan) [inline src/components/HomeComponent.js]: It shouldn't be here at all, my fault 👍

**Comment by kkorus** [comment]
I have renamed NavBarComponent to NavigationComponet, this may causing conflicts, rest looks good.

**Pull request merged**